### PR TITLE
Add durable_name to streaming subscription metrics

### PR DESCRIPTION
**Pull request by Will2817**
image

### CODE REVIEW

**Comment by Will2817** [comment]
One question I wasn't sure about was: Since the durable_name is in the queue_name label when the subscription is a queue group and durable, should the 
 queue_name label be split into durable_name and queue_name in this case?

File: collector/collector_test.go [hunk 1]
```diff
@@ -392,7 +392,8 @@ func TestStreamingSubscriptionsMetricLabels(t *testing.T) {
 	defer s.Shutdown()
 
 	queueName := "some-queue-name"
-	durableSubscriptionName := "some-durable-name"
+	durableSubscriptionName := "some-druable-name"
```
Comment (wallyqs) [inline collector/collector_test.go]:
```suggestion
	durableSubscriptionName := "some-durable-name"
```

**Comment by wallyqs** [comment]
@Will2817 thanks for PR!  splitting into queue_name and durable_name sounds like a good idea to me, would be useful to be able to query using different fields

**Comment by Will2817** [comment]
@wallyqs great, I've now updated the PR to also split the durable_name out from the queue_name when appropriate.

**Pull request merged**